Testing/Functional/SmokeTest/SmokeTest002.Tests.ps1

```diff
@@ -163,6 +163,9 @@ Describe -Tag "UI","Chrome" -Name "Test Report with <Browser> for $OrganizationN
                         }
                     }
                 }
+                elseif ($Table.GetProperty("id") -eq "license-info"){
+                    #Currently empty to determine if necessary and what to test in section
```

Comment:
suggest we add a test that the table is there in some form?

Reply:
Grant, not directly related to your suggestion but I wanted to ask you... The smoke test and the functional tests both run this same code that interrogates the HTML and I was thinking that it makes more sense to keep that stuff in the functional tests and have scope of the Smoke test be more about just making sure the tool does not crash. This would be to reduce redundancy and also to have clear goal for each type of test Smoke versus Functional. Thoughts?

Reply:
So I think the smoke test should have some structural formatting tests--not detailed but minimal.   I think arguably some of what is in there now for CAP could be simplified, but I was just asking for something that was basically like "there is a table and it has 3 columns" or the like.

Agree that detailed test cases belong in the functional tests.

Reply:
I added a test to ensure that the table has 3 columns headers and 3 columns of actual data, which should address this conversation

Reply:
I'm resolving so that we can merge the PR. Devesh added the requested code.